### ISSUE TITLE
Myth Unbound doesn't draw you a card when your commander is put back into command zone

### ISSUE THREAD

**Issue by alexander-novo** [magefree/mage#12209]
![image](https://github.com/magefree/mage/assets/11657716/0b7cd4ab-6736-49cb-a0f5-2322f4e13cbb)

![image](https://github.com/magefree/mage/assets/11657716/dbe62de7-1ef0-4a1f-8683-2b4208f6d4a8)

**Comment by alexander-novo** [comment]
Yeah it's currently only setup to look at moving from Battlefield -> Command Zone, despite the fact that [[Myth Unbound]] says "from anywhere"

https://github.com/magefree/mage/blob/d9fe633262f8c8797cf0dca448d404109b4c5120/Mage.Sets/src/mage/cards/m/MythUnbound.java#L44-L49

**Comment by xenohedron** [comment]
Yeah, the ZoneChangeAllTriggeredAbility that contains a fromZone parameter is misleading and should be removed. Any usage other than battlefield can't work with a FilterPermanent
